help/de/Help_ch_Account-Actions.xml

```diff
@@ -477,12 +484,13 @@
       </listitem>
 
       <listitem>
-        <para>Gibt es einen Anfangsbestand?
+        <para>Gibt es einen Anfangsbestand oder soll dieses Konto Anfangsbestand-Buchungen enthalten?
         </para>
       </listitem>
 
       <listitem>
-        <para>Soll das Konto Fremdwährungen oder Wertpapiere aufnehmen?
+        <para>Soll das Konto Fremdwährungen oder Wertpapiere aufnehmen? Dann benötigen Sie pro Währung/Aktie
+          vielleicht auch noch zusätzlich ein Konto für die Buchungen des Anfangsbestands.
```

Comment:
1. `/Aktie` raus WP-Buchungen sind immer zwischen einem WP und einer Währung. Falls man den Kaufpreis nicht zur Hand hat, mit 1 EUR einbuchen und nach Ermittlung berichtigen, (Kursdatenbank prüfen?)
2. Eigenkapital und seine temporären Varianten Income & Expense sollten grundsätzlich in der Basiswährung geführt werden. Sonst kann man in Teufels Küche kommen. Fremdwährungen werden hier aber vom Programm nicht aktiv unterbunden.

Reply:
Ich habe Punkt 2 als `<tip>`ergänzt. Warum sind Erträge und Aufwand temporäre Varianten von Eigenkapital?

Reply:
Weil der Saldo im Jahresabschluß grundsätzlich ins Eigenkapital abgeschlossen wird.

Bei AGs kann ein Teil auch als Dividende ausgeschüttet werden.
Bei Fonds gibt es oft die klare Unterscheidung (gewinn-) ausschüttend oder kumulierend.